dataactbroker/check_status.yml

```diff
@@ -0,0 +1,127 @@
+swagger: '2.0'
```

Comment:
I don't think we have a process around swagger yet.  This was what we created to document this route, if there's somewhere else we should put these, I can put it there instead.  I put it here because this is where the current route documentation is in the readme.  Paging @nmonga91, what was done with the swagger from last sprint?